CMakeLists.txt

```
@@ -2,10 +2,15 @@
 # Copyright (C) 2022 Whitney Armstrong, Sylvester Joosten, Wouter Deconinck
 
 cmake_minimum_required(VERSION 3.12)
+
 project(EDM4EIC
-  VERSION 1.1.0
   LANGUAGES CXX)
 
+SET( ${PROJECT_NAME}_VERSION_MAJOR 2 )
+SET( ${PROJECT_NAME}_VERSION_MINOR 0 )
```

Safe to put this to the next release version
```suggestion
SET( ${PROJECT_NAME}_VERSION_MINOR 1 )
```